src/backend/distributed/commands/citus_version.c

```diff
@@ -0,0 +1,36 @@
+/*-------------------------------------------------------------------------
+ *
+ * citus_version.c
+ *	  Routines related to the version of the Citus binaries
+ *
+ * Copyright (c) 2012-2016, Citus Data, Inc.
+ *
+ *-------------------------------------------------------------------------
+ */
+
+#include "postgres.h"
+
+#include "fmgr.h"
+
+#include "citus_version.h"
+#include "utils/builtins.h"
+
+
+/* exports for SQL callable functions */
+PG_FUNCTION_INFO_V1(citus_running_version);
+
+#if !defined(CITUS_BINARY_VERSION)
```

Comment:
If we make it a define, can't I just change our scripts to provide it at build time? Using a `-D` option to compilation or something?

Don't worry about that right now, just do the patch version and we're good.